minigalaxy/launcher.py

```diff
@@ -142,7 +139,15 @@ def get_windows_exe_cmd(game, files):
         filename = os.path.splitext(os.path.basename(executables[0]))[0] + '.exe'
         exe_cmd = [get_wine_path(game), filename]
 
-    return exe_cmd
+    # Backwards compatibility with windows games installed before installer fixes.
+    # Will not fix games requiring registry keys, since the paths will already
+    # be borked through the old installer.
+    gamelink = os.path.join(prefix, 'dosdevices', 'c:', 'game')
+    if not os.path.exists(gamelink):
+        os.makedirs(os.path.join(prefix, 'dosdevices', 'c:'))
```

Comment:
Because this quickly becomes a maintenance and testing hell. There's also the issue of cyclomatic complexity. I always try to avoid deep nesting if simple ifs can instead be used to make sure a certain requirement is given before actually using a common piece of code.

Reply:
Makes sense, thanks for clarifying.